api/controller/protected_domain.py

```diff
@@ -80,3 +82,40 @@ def list_protected_domains():
     ### Click  `Try it out` to use the endpoint
     """
     return protected_domain_service.list_protected_domains()
+
+
+@protected_domain_router.delete(
+    "/{domain}",
+    dependencies=[Security(secure_endpoint, scopes=[Action.DATA_ADMIN.value])],
+)
+def delete_protected_domain(domain: str, response: Response):
+    """
+    ## Delete protected domain
+
+    Use this endpoint to delete a specific protected domain linked to a domain. If there is no protected domain with this name it
+    will throw an error, likewise if the protected domain currently contains any datasets it will throw an error as the protected
+    domain is not empty.
+
+    When a valid empty protected domain is deleted, a success message will be displayed.
+
+    ### Inputs
+
+    | Parameters       | Usage               | Example values   | Definition                       |
+    |------------------|---------------------|------------------|----------------------------------|
+    | `domain`         | URL Parameter       | `land`           | The name of the protected domain |
+
+    ### Accepted permissions
+
+    In order to use this endpoint you need the `DATA_ADMIN` permission
+
+    ### Click  `Try it out` to use the endpoint
+    """
+    subjects_list = subject_service.list_subjects()
+    user_subjects_list = [
+        subject["subject_id"] for subject in subjects_list if subject["type"] == "USER"
```

Comment:
Why is this subject_id = "USER"? Surely we want to change the permissions for the clients too?